### PR TITLE
virsh_domstate: configure less guest memory for coredump testcase

### PR DESCRIPTION
**Pull request by balamuruhans**
guest with huge memory takes more time for dumping core which timeout
the testcase and cause error, instead configuring less memory would
ensure test to work properly

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by balamuruhans** [comment]
This PR depends on https://github.com/avocado-framework/avocado-vt/pull/1340

File: libvirt/tests/src/virsh_cmd/monitor/virsh_domstate.py [hunk 1]
```diff
@@ -102,6 +102,16 @@ def run(test, params, env):
     vmxml = vm_xml.VMXML.new_from_inactive_dumpxml(vm_name)
     backup_xml = vmxml.copy()
 
+    # Let's have guest memory less so that dumping core takes
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/monitor/virsh_domstate.py]: suggest move those implementation to line 124 to make it more understandable

**Comment by balamuruhans** [comment]
@chunfuwen made the changes as suggested.

**Comment by balamuruhans** [comment]
Fixed another issue observed with new commit, the guest's core file generated on crash dumped with file name having guest name, but if guest name have more than 20 characters, it clips it causing the tests to error out.

**Comment by balamuruhans** [comment]
Thanks @chunfuwen!

**Pull request merged**